linux_os/guide/system/software/disk_partitioning/encrypt_partitions/rule.yml

```diff
@@ -94,3 +94,8 @@ ocil: |-
     are not required to use disk encryption and are not a finding.
 
 platform: machine
+
+fix: |-
+    Configure {{{ full_name }}} to prevent unauthorized modification of all information at rest by using disk encryption.
+
+    Encrypting a partition in an already installed system is more difficult, because existing partitions will need to be resized and changed. To encrypt an entire partition, dedicate a partition for encryption in the partition layout.
```

Comment:
Should we place the second sentence on a new line to help control the line length?